Assets/Mirror/Editor/Weaver/Weaver.cs

```diff
@@ -31,6 +31,22 @@ internal class Weaver
         // multi threaded logging.
         public Logger Log;
 
+        // remote actions now support overloads,
+        // -> but IL2CPP doesnt like it when two generated methods
+        // -> have the same signature,
+        // -> so, append the signature to the generated method name,
+        // -> to create a unique name
+        // fixes https://github.com/vis2k/Mirror/issues/3060
```

Comment:
please add a method + generated name example to comments for future reference :)